### DISCUSSION TITLE
8 TV Cadiz [ES]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#380] [Show and tell]
### Channel ID

8TVCadiz.es

### Stream URL

https://5940924978228.streamlock.net/8289/smil:8289.smil/playlist.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Stream does not work.

**Comment by BellezaEmporium** [comment]
May be Not 24/7.

**Comment by freearhey** [comment]
https://5940924978228.streamlock.net/8289/smil:8289.smil/playlist.m3u8 (offline)
